### DISCUSSION TITLE
Best way for minimal backup size ? (Regenerate thumbs)

### DISCUSSION THREAD

**Discussion by kevin-snt** [LycheeOrg/Lychee#2032] [Q&A] [answered]
Hello, I read the doc and read several topics but I couldn't find anything specific on this subject.

I want to make a backup of my Lychee project, particularly regarding the photos, and I thought to myself that it was a bit stupid to backup all the formats, knowing that they are generated automatically.

I saw that there was the command "lychee:generate_thumbs", if I only backup the "original" folder, will this command allow everything to be regenerated?

I was surprised to see that my original folder was smaller than the "big" folder, and therefore I would have wanted to confirm which folder to backup :D

Thanks for all the work on this project, it’s great!

**Comment by ildyria** [comment, marked as answer]
Sadly  you need to store **both** `big` and `originals`. The reason being that `big` was the old name in previous Lychee v3 and some older v4 versions. We changed the name to `original` to reflect what those actually contain.

In addition to that you may need to remove all the `small`, `thumbs`, `medium` from your SizeVariants table. :)

**Reply by kevin-snt** [reply]
Thank you for your quick reply

So I can just backup the "big" and "original" folder (and of course the db etc), and to restore the data I will just have to delete the lines in SizeVariants and run the thumbs generation command

I would try it on a local env
I will also see if I can migrate the "big" folder into "original"

**Reply by ildyria** [reply]
You can most likely migrate some "big" into "original" but that will force you to rename some of your size variants url in the DB.
